rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2509,6 +2510,14 @@ def test_upgrade_db_31_to_32(user_data_dir):  # pylint: disable=unused-argument
     cursor.execute('SELECT COUNT(*), timestamp FROM history_events WHERE subtype="staking receive asset" AND type="unstaking"')  # noqa: E501
     assert cursor.fetchone() == (1, expected_timestamp)
 
+    # check that you cannot add blockchain column in xpub_mappings
```

Comment:
no no no absolutely not. As discussed you **can not** modify a DB upgrade that has already been released to the public. Make a new one and test it properly. You will also need to make a new empty test v32 DB and populate it with xpubs and see they get upgraded properly.